### DISCUSSION TITLE
Raven scanner - PDFs with OCR data from scanner

### DISCUSSION THREAD

**Discussion by SkyMule** [ciur/papermerge#454] [Q&A]
Ref: [https://www.raven.com/products/raven-scanner-original](url)

This brand/model scanner has its own OCR engine...

_AI-Powered OCR (optical character recognition) automatically turns your documents into searchable PDF files, making it easy to to find the document you are looking for, including both printed and handwritten text._

I know the above isn't a technical description.

My question: How can I determine if Papermerge can (or should) utilize the OCR  data generated by the scanner?

It's not a big deal... just curious.

I haven't purchased the scanner yet... but the company (based in Texas, USA) could probably send me a sample scanner-generated PDF with OCR.

Thanks!

Lee

**Comment by ciur** [comment]
Question is very blurry, not sure what to say here.

**Reply by ciur** [reply]
I've checked Raven scanner, looks really great! Thanks for sharing the link!
